core-bundle/src/Resources/contao/controllers/FrontendIndex.php

```diff
@@ -216,6 +216,19 @@ public function renderPage($pageModel)
 			}
 		}
 
+		$time = Date::floorToMinute();
+
+		// Trigger the 404 page if the page is not published and the front end preview is not active (see #374)
+		if (!$objPage->published || ($objPage->start !== '' && $objPage->start > $time) || ($objPage->stop !== '' && $objPage->stop < $time))
```

Comment:
As this is legacy code, and a legacy file, I would just use `BE_USER_LOGGED_IN` 👀 😁

Reply:
Agreed. Changed in a7457132246416d5038dfe11f03ba3665d2d7238.